source/_integrations/input_number.markdown

```diff
@@ -14,7 +14,7 @@ ha_integration_type: helper
 
 The `input_number` integration allows the user to define values that can be controlled via the frontend and can be used within conditions of automation. The frontend can display a slider, or a numeric input box. Changes to the slider or numeric input box generate state events. These state events can be utilized as `automation` triggers as well.
 
-The preferred way to configure an input number is via the user interface at **{% my helpers title="Settings > Devices & Services > Helpers" %}**. Click the add button and then choose the **{% my config_flow_start domain=input_number title="Number" %}** option.
+The preferred way to configure an input number is via the user interface at **{% my helpers title="Settings > Devices & Services > Helpers" %}**. Click the add button and then choose the **{% my config_flow_start domain="input_number" title="Number" %}** option.
```

Comment:
Why don't we use the page `ha_domain` and `title` here?

Reply:
Because then it would pull the title "Input Number". And whoever wrote this first decided that "Number" would be a better-suited title. I did not want to change the content. Just fix the error. I noticed this in other helpers, the team "Input" is not shown in the title.

Reply:
"Number" sounds more like natural language to me. Unless we need to make the distinction? Do we have an output number?

Reply:
Shouldn't we change the page title as well in that case?

Reply:
They are different and deliberate.

In the UI there is more context (already on the helpers page). So it is a "number" helper.

But on the documentation, that context is not clear in the title.